docs/dxp/7.x/en/site-building/site-appearance/site-appearance.md

```diff
@@ -12,46 +12,33 @@ The design and functionality of a Liferay site can be adjusted and customized th
 
 ## Master Page Templates
 
-Coming soon!
-<!-- TODO:
+[Master Page Templates](../creating-pages/defining-headers-and-footers/master-page-templates.md) allow you to define elements for all of your pages using Fragments, especially headers and footers. You can use them to conform any number of pages on your Site to the look and feel you design.
 
-[Screenshot of creating a master page template]
-
-Introduce concept of Master Page Template, as well as other Page Templates (and the distinction between the two). Link to respective articles -->
+![Master Page Templates are a simple way to define elements common to all of your pages, such as headers and footers.](./site-appearance/images/01.png)
 
 ## Style Books
 
-[Screenshot of creating a style book?]
-
-Introduce Style Books, note a few really general use cases to give the idea. Note that all the levels "above" (page templates, fragments, etc.) all use these in their displays
-
-## Widget/Fragment Appearance
-
-[Maybe show a considerably customized-looking widget/fragment?]
+[Style Books](./style-books/using-a-style-book-to-standardize-site-appearance.md) are sets of visual standards that apply to a Site, such as standard colors and spacing. This allows you to design every page's layout independently while maintaining a consistent experience between each of them.
```

Comment:
Whatever we write here, it should just be a very brief description taken from the linked page. I'm assuming that's the case here